### ISSUE TITLE
publish fonts to Google Fonts

### ISSUE THREAD

**Issue by ribrdb** [KaTeX/KaTeX#3799]
Would it be possible to submit the katex fonts to Google Fonts?
https://github.com/google/fonts/blob/main/CONTRIBUTING.md

I'd like to use katex in a Looker Studio Community Visualization, but they have a csp policy that only allows fonts from https://fonts.googleapis.com/ and https://fonts.gstatic.com/

**Comment by shindodkar** [comment]
https://fonts.googleapis.com/ and https://fonts.gstatic.com/  -  Those two websites showing error. @ribrdb

**Comment by ribrdb** [comment]
Yes, those are not valid urls. I am trying to use katex from a website whose CSP policy only allows fonts hosted on one of those two domains. The first link contains instructions to add a font to be hosted on Google Fonts, so that Google will serve it from fonts.googleapis.com.

**Comment by Anamika1-cpu** [comment]
Assign this to me
